_data-prepper/pipelines/pipelines.md

```diff
@@ -58,7 +58,7 @@ When any component of a pipeline fails and is unable to send an event, the sourc
 
 As of Data Prepper 2.2, only the `s3` source and `opensearch` sink support E2E acknowledgments. 
 
-## Conditional routing
+## Conditional routing<a name="conditional_routing"></a>
```

Comment:
Why are we adding an alternate `a name` here? The link above will work with the reformatting specified in the comment above.

Reply:
Thanks, I thought that did not work, but I will update.